app/queries/admin/users_query.rb

```diff
@@ -26,6 +31,18 @@ def self.search_relation(relation, search)
       relation.where(QUERY_CLAUSE, search: "%#{search.strip}%")
     end
 
+    def self.filter_organization_memberships(relation:, organizations:)
+      org_conditions = []
+      organizations.each do |org_id|
+        org_conditions << "organization_memberships.organization_id = #{org_id}"
+      end
+
+      relation.joins(
+        "INNER JOIN organization_memberships ON organization_memberships.user_id = users.id
```

Comment:
One problem with the inner join, is that if a user belongs to two organizations, and we filter for both of those organizations, we'll end up with two user rows for the same user:

One row for the first organization and another for the other organization.